src/constants.ts

```diff
@@ -46,3 +46,20 @@ export const CONST_SETTINGS_TAB_SLUGS = [
   "notifications",
   "manage-account",
 ] as const;
+export const CONST_DATA_BROKER_PROFILE_DETAIL_ALLOW_LIST = [
+  "first_name",
+  "last_name",
+  "middle_name",
+  "first_names",
+  "last_names",
+  "middle_names",
+  "phone_numbers",
+  "addresses",
+] as const;
+export const CONST_DATA_BROKER_PROFILE_DETAIL_LIMITS = {
```

Comment:
How are these limits enforced? We found in the past that is was insufficient to do this on the front-end, and we ended up enforcing this at the database level using transactions with a strong isolation level https://github.com/mozilla/blurts-server/blob/d92af9a396ff843184a64f7af08a7bfecfe43372/src/db/tables/emailAddresses.ts#L49

Reply:
The database entries are checked for their limits before writing the updates [here](https://github.com/mozilla/blurts-server/pull/5660/files#diff-a3bf4c5bb3b191682ce0a888e2ceefcd2001f3dc4939b983d70fe2623154e65dR52-R65). Since this still leaves a window of opportunity to insert more entries than the limit allows for setting the transaction isolation level is a good idea. Added in [27b9cc8](https://github.com/mozilla/blurts-server/pull/5660/commits/27b9cc8d553c46452fef17592980941855ab4508).